### ISSUE TITLE
Vi cursor doesnt work with alacritty

### ISSUE THREAD

**Issue by adamijak** [fish-shell/fish-shell#7564]
fish, version 3.1.2

Linux ThinkArch 5.9.14-arch1-1 #1 SMP PREEMPT Sat, 12 Dec 2020 14:37:12 +0000 x86_64 GNU/Linux

TERM=alacritty

```fish
#
# ~/.config/fish/config.fish
#
fish_vi_key_bindings insert
fish_vi_cursor
set fish_cursor_default block
set fish_cursor_insert line
set fish_cursor_replace_one underscore
set fish_cursor_visual block

if status --is-interactive
	thefuck --alias | source
	starship init fish | source
end
```

**Comment by faho** [comment]
Fixed with 3c6c8d542410cd84b8249c84a58ec28db71fba06.

**Comment by faho** [comment]
Duplicate of #7443
